django/publicmapping/redistricting/templates/editplan.html

```diff
@@ -351,7 +351,77 @@ <h2>{% blocktrans %}3. Paste selected {{ body_member_long_label }} into working
+                        <option value="Bradford">Bradford</option>
+                        <option value="Bucks">Bucks</option>
+                        <option value="Butler">Butler</option>
+                        <option value="Cambria">Cambria</option>
+                        <option value="Cameron">Cameron</option>
+                        <option value="Carbon">Carbon</option>
+                        <option value="Centre">Centre</option>
+                        <option value="Chester">Chester</option>
+                        <option value="Clarion">Clarion</option>
+                        <option value="Clearfield">Clearfield</option>
+                        <option value="Clinton">Clinton</option>
+                        <option value="Columbia">Columbia</option>
+                        <option value="Crawford">Crawford</option>
+                        <option value="Cumberland">Cumberland</option></option>
+                        <option value="Dauphin">Dauphin</option>
+                        <option value="Delaware">Delaware</option>
+                        <option value="Elk">Elk</option>
+                        <option value="Erie">Erie</option>
+                        <option value="Fayette">Fayette</option>
+                        <option value="Forest">Forest</option>
+                        <option value="Franklin">Franklin</option>
+                        <option value="Fulton">Fulton</option>
+                        <option value="Greene">Greene</option>
+                        <option value="Huntingdon">Huntingdon</option>
+                        <option value="Indiana">Indiana</option>
+                        <option value="Jefferson">Jefferson</option>
+                        <option value="Juniata">Juniata</option>
+                        <option value="Lackawanna">Lackawanna</option>
+                        <option value="Lancaster">Lancaster</option>
+                        <option value="Lawrence">Lawrence</option>
+                        <option value="Lebanon">Lebanon</option>
+                        <option value="Lehigh">Lehigh</option>
+                        <option value="Luzerne">Luzerne</option>
+                        <option value="Lycoming">Lycoming</option>
+                        <option value="McKean">McKean</option>
+                        <option value="Mercer">Mercer</option>
+                        <option value="Mifflin">Mifflin</option>
+                        <option value="Monroe">Monroe</option>
+                        <option value="Montgomery">Montgomery</option>
+                        <option value="Montour">Montour</option>
+                        <option value="Northampton">Northampton</option>
+                        <option value="Northumberland">Northumberland</option>
+                        <option value="Perry">Perry</option>
+                        <option value="Philadelphia">Philade</option>lphia
```

Comment:
Some of these labels are truncated. I noticed at least this one and Westmoreland.

Reply:
Ugh, good catch, thanks! I think I was lulled into a false sense of security by not seeing any linter errors, but that was just because ALE doesn't have a linter for Django templates. I've fixed those two and also removed a double `</option> up on 376.